srcpkgs/system76-keyboard-configurator/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'system76-keyboard-configurator'
+pkgname=system76-keyboard-configurator
+version=1.3.10
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config"
+makedepends="gtk+3-devel hidapi-devel eudev-libudev-devel glib-devel"
+short_desc="Tool for configuring System76 keyboards"
+maintainer="Henri Koski <henri.t.koski@gmail.com>"
+license="GPL-3.0-only"
+homepage="https://github.com/pop-os/keyboard-configurator"
+distfiles="https://github.com/pop-os/keyboard-configurator/archive/refs/tags/v1.3.10.tar.gz"
```

Comment:
use `$version` instead of hardcoding it in the url

Reply:
Fixed